src/main/java/org/fcrepo/importexport/exporter/Exporter.java

```diff
@@ -244,6 +247,8 @@ private void export(final URI uri) {
                 exportBinary(uri, describedby, external);
             } else if (linkHeaders.contains(containerURI)) {
                 exportDescription(uri);
+                // Export versions for this container
+                exportVersions(uri);
```

Comment:
Yes, this was somewhat unfortunate but intentional.  For binaries `exportVersions` has to be called after a check of `config.isIncludeBinaries`, which happens inside of `exportBinary`.  But I couldn't mirror that arrangment with containers since `exportDescription` is called by binaries as well (having it in there caused issues with exporting the wrong URI for binaries).  

I could have moved the `isIncludeBinaries` check out of exportBinary, but that seemed out of line with how other methods were doing precondition checks.  If you'd prefer I can move it though.